config/recommender.json

```diff
@@ -0,0 +1,11 @@
+{
+    "recommender": {
+        "content-based": "LDA",
+        "collaborative-filtering": "ALS",
```

Comment:
ALS is the only method we are gonna use anyways, I think it is useless to put it in the configuration.